tpm/tpm_test.go

```diff
@@ -232,6 +232,33 @@ func TestResizeableSlice(t *testing.T) {
 	}
 }
 
+func TestNVDefineSpace(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := tpmutil.Handle(0x40000001)
+	permissions := []uint32{nvPerWriteSTClear}
+	length := uint32(54)
+	ownerAuth := getAuth(ownerAuthEnvVar)
+	_, err := NVDefineSpace(rwc, index, length, permissions, ownerAuth)
```

Comment:
Please fold this into the if statement if the return value isn't going to be checked. Is there something that can be checked here?

Reply:
The check for a successful operation is done in submitTPMRequest-Function.
I could parse the the response from the tpm, but I cant see the benefit here.